cloth-diapers/wash-routine.md

```diff
@@ -0,0 +1,139 @@
+# Wash routine
+
+> The wash routine for cloth diapers is the **main source** of _every_ cloth diaper issue from smells to rashes to leaks.
+
+The cloth diaper groups online are ***filled*** with people bemoaning issues with their cloth diapers that turn into nothing but 'let's trouble shoot your wash routine' posts.
```

Comment:
The sentiment of this sentence comes across as 🤦‍♀️ when it could be written in a less 🙄 style ...
e.g:
```suggestion

Cloth diaper forums/groups online are ***filled*** with people bemoaning issues with their cloth diapers. Most of the time the issue is their wash routine e.g. machine cycle, temperature, detergent, drying, etc. This is all easy to fix once you understand the process.
```